Dshop/apps/products_catalogue/views.py

```diff
@@ -1,3 +1,49 @@
-#from django.shortcuts import render
+from django.views import View
+from lxml import etree
 
-# Create your views here.
+from django.http import HttpResponse
+
+from .models import Product
+
+
+class ProductListView(View):
+    model = Product
+
+    def get_queryset(self):
+        queryset = self.model.objects.filter(is_active=True)
+        return queryset
+
+    def generate_xml_file_for_ceneo(self, products):
+        root = etree.Element('offers', xmlns_xsi="http://www.w3.org/2001/XMLSchema-instance", version="1")
```

Comment:
```suggestion
    def generate_xml_for_ceneo(self, products):
        root = etree.Element('offers', xmlns_xsi="http://www.w3.org/2001/XMLSchema-instance", version="1")
```
Let's make it return xml only, not http response

Reply:
More details below.